src/main/java/com/jm/lms/studentms/controller/EnrollCourseController.java

```diff
@@ -0,0 +1,39 @@
+package com.jm.lms.studentms.controller;
+
+import java.util.List;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.jm.lms.studentms.model.EnrollCourse;
+import com.jm.lms.studentms.service.EnrollCourseService;
+
+@RestController
+@RequestMapping("api/v1/lms/Course")
```

Comment:
class level uri should be like api/v1/lms/enroll

Reply:
Done sir